systemd/google-cloud-ops-agent-fluent-bit.service.d/directories.conf

```diff
@@ -1,5 +1,5 @@
 # For distros with systemd prior to version 240:
 [Service]
-Environment=RUNTIME_DIRECTORY=%t/google-cloud-ops-agent-fluent-bit STATE_DIRECTORY=/var/lib/google-cloud-ops-agent/fluent-bit LOGS_DIRECTORY=/var/log/google-cloud-ops-agent/subagents
-ExecStartPre=/bin/mkdir -p ${RUNTIME_DIRECTORY} ${STATE_DIRECTORY} ${LOGS_DIRECTORY}
+Environment=RUNTIME_DIRECTORY=%t/google-cloud-ops-agent-fluent-bit STATE_DIRECTORY=/var/lib/google-cloud-ops-agent/fluent-bit LOGS_DIRECTORY=/var/log/google-cloud-ops-agent
+ExecStartPre=/bin/mkdir -p ${RUNTIME_DIRECTORY} ${STATE_DIRECTORY} ${LOGS_DIRECTORY}/subagents
```

Comment:
Should this also create `${LOGS_DIRECTORY}` by itself before creating the `..../subagents` subfolder ?

Reply:
The -p flag creates the parent aswell.